frontend-nextjs/README.md

```
@@ -53,7 +53,7 @@ URLクリックでブラウザ起動
 ```
 
 ## Dockerコンテナの設定
-- `./frontend-react/Dockerfile.frontend-nextjs`のファイル作成
+- `./frontend-nextjs/Dockerfile.frontend-nextjs`のファイル作成
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)